Model/Service/Stock/Provider/DefaultStockProvider.php

```diff
@@ -117,4 +120,20 @@ private function getStockStatuses(// @codingStandardsIgnoreLine
             StockRegistryProvider::DEFAULT_STOCK_SCOPE
         )->getItems();
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function getInStockSkusByIds(array $productIds, Website $website)
+    {
+        $stockItems = $this->getStockStatuses($productIds, $website);
+        $skus = [];
+        /** @var Status $stockItem */
+        foreach ($stockItems as $stockItem) {
+            if ($stockItem->getStockStatus() == 1) {
```

Comment:
Would be better to use the `STATUS_IN_STOCK` constant from `Magento\CatalogInventory\Api\Data\StockStatusInterface`

Reply:
👍